.travis.yml

```diff
@@ -5,13 +5,14 @@ install:
 - npm install
 - npm install -g codecov
 - npm install -g nyc
+- mkdir site

```

Comment:
I personally like abstracting the CICD file to just run make commands. That way you are more adjusting only the stages themselves in this file. Just something to think about later I suppose.

Reply:
For sure. This was a test of mixing the mkdocs with `npm run`, put that in as a placeholder. Forgot to remove it.